bin/GenValidation/updated_validation_cards/wjets_fxfx/w012jfxfx_inc/wellnu012j_5f_NLO_FXFX_madspin_card.dat

```diff
@@ -0,0 +1,27 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
```

Comment:
I think this is not needed for onlyhelicity mode. Also the decay line has no impact. @sihyunjeon confirmed this.

Reply:
updated, removed these lines